app/components/views/GetStartedPage/hooks.js

```diff
@@ -208,9 +208,18 @@ export const useGetStarted = () => {
         if (synced === true) {
           send({ type: "GO_TO_HOME_VIEW" });
         }
-        if (context.isSPV) {
+        if (isSPV) {
           return startSPVSync(passPhrase)
-            .then(() => send({ type: "GO_TO_HOME_VIEW" }))
+            .then(() => {
+              if (isPrivacy) {
+                // if recoverying a privacy wallet, we go to settingMixedAccount
+                // state, so the user can set a mixed account based on their
+                // coinjoin outputs.
+                // This state should only be achievable if recoverying wallet.
```

Comment:
This state shouldn't be only achievable when recoverying a wallet, as the user may pass it and may want to do it later, but I am approving this PR, as this is true for this release, but we should allow it if one passes it, by mistake or whatever other reason.

Reply:
You are right and this was completely deleted in #3033 as it will run on each startup now!